web/src/main/java/no/nav/foreldrepenger/web/app/tjenester/behandling/ytelsefordeling/YtelseFordelingDtoTjeneste.java

```diff
@@ -12,13 +12,16 @@
 import no.nav.foreldrepenger.behandlingslager.behandling.ytelsefordeling.AvklarteUttakDatoerEntitet;
 import no.nav.foreldrepenger.behandlingslager.behandling.ytelsefordeling.YtelseFordelingAggregat;
 import no.nav.foreldrepenger.behandlingslager.fagsak.FagsakRelasjonRepository;
+import no.nav.foreldrepenger.domene.uttak.ForeldrepengerUttak;
 import no.nav.foreldrepenger.domene.uttak.ForeldrepengerUttakTjeneste;
 import no.nav.foreldrepenger.domene.uttak.UttakOmsorgUtil;
 import no.nav.foreldrepenger.domene.ytelsefordeling.YtelseFordelingTjeneste;
 
 @ApplicationScoped
 public class YtelseFordelingDtoTjeneste {
 
+    private static final LocalDate DATO_FOR_TOMT_UTTAK = LocalDate.of(2199, 12, 31);
```

Comment:
Skal dette være uendelig? Burde man da heller brukt Tid.TidenesEnde?  Om det ikke er uendelig, kanskje greit å forklare hvorfor 2199, var ikke åpenbart for meg i alle fall.

Reply:
Datoen folketrygden ikke finnes lenger?🗡️

Reply:
Tenkte først på tidenes ende. Ble litt usikker på hva frontend vil si til den datoen. Reverter til tidenes ende